src/apps/common/submodules/callerId/callerId.js

```diff
@@ -12,11 +12,24 @@ define(function(require) {
 			'common.callerId.renderPopup': 'callerIdEdit'
 		},
 
+		/**
+		 * Show Caller ID edit options for a phone number
+		 * @param  {Object}   args
+		 * @param  {Object}   [args.accountId]          Account ID
+		 * @param  {String}   args.phoneNumber          Phone number whose caller ID data will
+		 *                                              be edited
+		 * @param  {Object}   [args.callbacks]          Callback functions
+		 * @param  {Function} [args.callbacks.success]  Success callback
+		 * @param  {Function} [args.callbacks.error]    Error callback
+		 */
 		callerIdEdit: function(args) {
 			var self = this,
 				argsCommon = {
-					success: function(dataNumber) {
-						self.callerIdRender(dataNumber, args.accountId, args.callbacks);
+					success: function(numberData) {
+						self.callerIdRender(_.assign({
```

Comment:
Using `lodash#assign` here can be misleading since you just want to add `success`/`error` to this object I believe, Unless you want to override `numberData`?
```suggestion
						self.callerIdRender(_.merge({
```